private/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package private contains helpers private to github.com/containers/image, with no intent or commitment to make them useful for any other purpose.
+// Do not use outside!
+package private
```

Comment:
Yeah, kind of; they would obviously not be actually inheriting from each, other, but “I am the OpenShift transport, I use a dockerImageSource for some operations, and I import docker/policyconfiguration for some others” is a readable story.